vnext/Microsoft.ReactNative.SharedManaged/AttributedViewManager.cs

```diff
@@ -4,15 +4,19 @@
 using System;
 using System.Collections.Generic;
 using System.Reflection;
-using Windows.UI;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Media;
 
 using Microsoft.ReactNative.Bridge;
 
 namespace Microsoft.ReactNative.Managed
 {
-  internal abstract class AttributedViewManager<T> : IViewManager where T : FrameworkElement, new()
+  internal abstract class AttributedViewManager<T> :
+    IViewManager,
+    IViewManagerWithExportedViewConstants,
+    IViewManagerWithNativeProperties,
+    IViewManagerWithCommands
```

Comment:
I have mixed feelings about this pattern. It makes implementing both more simple and more fussy at the same time. I think on the whole it's a net positive but it's probably worth floating some discussion on API review. There may be some other patterns we can adopt.